README.md

```diff
@@ -25,12 +30,106 @@ Social, Advertising, or Analytics categories as described
 This remapping occurs at the time of list creation, so the Social, Analytics,
 and Advertising lists consumed by Firefox will contain these domains.
 
-### Entity list
-Tracking protection technically works by blocking loads from blocked domains. But the Entity List conceptually changes it, so that it is no longer about domains, but about the companies. If you are visiting a website, engaged 1-on-1 with them, Tracking Protection will block the other companies who the user may not realize are even present and didn't explicitly intend to interact with.
-
-Tracking Protection blocks loads on the blacklist when they are third-party. The Entity list whitelists different domains that are wholly owned by the same company. For example, if abcd.com owns efgh.com and efgh.com is on the blacklist, it will not be blocked on abcd.com. Instead, efgh.com will be treated as first-party on abcd.com, since the same company owns both. But since efgh.com is on the blacklist it will be blocked on other third-party domains that are not all owned by the same parent company.
-
-## Updating
+Firefox consumes the list as follows:
+* **Tracking**: anything in the Advertising, Analytics, Social, Content, or
+    Disconnect category. Firefox ships two versions of the tracking lists: the
+    "Level 1" list, which excludes the "Content" category, and the
+    "Level 2" list which includes the "Content" category.
+* **Cryptomining**: anything in the Cryptomining category
+* **Fingerprinting**: anything in the Fingerprinting category. By default,
+    ETP's fingerprinting blocking only blocks _Tracking Fingerprinters_, that
+    is domains which appear in both the Fingerprinting category and one of the
+    Tracking categories.
+
+#### `disconnect-entitylist.json`
+
+A version controlled copy of Disconnect's
+[list of entities](https://github.com/disconnectme/disconnect-tracking-protection/blob/master/entities.json).
+ETP classifies a resource as a tracking resource when it is present on
+blocklist and loaded as a third-party. The Entity list is used to whitelist
+third-party subresources that are wholly owned by the same company that owns
+the top-level website that the user is visiting. For example, if abcd.com owns
+efgh.com and efgh.com is on the blacklist, it will not be blocked on abcd.com.
+Instead, efgh.com will be treated as first party on abcd.com, since the same
+company owns both. But since efgh.com is on the blacklist it will be blocked on
+other third-party domains that are not all owned by the same parent company.
+
+## Other lists
+
+In addition, Mozilla maintains a several lists for Firefox-specific features
+and experiments. The lists currently in active use are:
+* `social-tracking-protection-blacklist.json`: a subset of social trackers from
+    Disconnect's blocklist. This list is used to identify "social media"
+    trackers within Firefox's UI. All of the origins on this list should also
+    be included in Disconnect's `disconnect-blacklist.json` list.
```

Comment:
```suggestion
    be included in "Social" category of Disconnect's `disconnect-blacklist.json` list.
```

Reply:
I'm not sure this is true right now, and I don't think we intend to restrict ourselves to the Social category. Have you heard otherwise?

Reply:
Oh, I was under the impression that all the domains on this is already listed under the "Social" category in the Disconnect's lists. Isn't that what you meant on the [sentences](https://github.com/mozilla-services/shavar-prod-lists/pull/97/files#diff-04c6e90faac2675aa89e2176d2eec7d8R61-R62) before? I was reiterating that.

Reply:
I've updated this to "a subset of trackers ..."